RELEASE.md

```diff
@@ -0,0 +1,13 @@
+# Instructions for making a release
+
+1. Change the version number in package.json. on the command line, in the package root directory, run the following command, replacing <update_type> with one of the semantic versioning release types (prerelease, prepatch, preminor, premajor, patch, minor, major):
+
+npm version <update_type> --preid pre --no-git-tag-version
```

Comment:
you could put this in triple backticks (there's no way for me to suggest that on github since it also uses triple backticks for suggestions :laughing:)

Reply:
I added in the ticks